documents/charter.md

```diff
@@ -16,28 +16,28 @@ Application providers are even designing algorithms to detect the presence of su
 Instead, it would be beneficial, for both the application provider and network operator, to signal network attributes to the application to self-adapt its video traffic to conform to the specified characteristics.
 The application provider has the ability to measure end user QoE and therefore can self-adapt with QoE feedback.
 
-The Secure Communication of Network Properties (SCONEPRO) Working Group's primary objective is to specify an on-path protocol for securely communicating network properties to clients relevant to a given application, such as the maximum achievable bandwidth for a video. 
+The Secure Communication of Network Properties (SCONEPRO) Working Group's primary objective is to specify an on-path protocol for securely communicating network properties to clients relevant to a given application, such as the maximum achievable bandwidth for a video.
 - The working group will initially focus on a solution that communicates the maximum achievable bandwidth for a video delivered from a server to a client, using QUIC connections carrying the application signaling traffic.
-- Work to support TCP or other transport protocols may be considered later in the working group, however, these considerations shouldn't distract from support for video over QUIC. 
+- Work to support TCP or other transport protocols may be considered later in the working group, however, these considerations shouldn't distract from support for video over QUIC.
 - Further use cases may be considered later in the working group, however, it is not assumed that future use cases must or can be addressed by the same protocol. In essence, any protocol specified by the working group should be tailored to solve a specific use case.
 
 The properies of this mechanism are as follows:
 
-1. Associativity with an application. 
+1. Associativity with an application.
 The network properties must be associated with a given application traversing the network, for example a video playback.
-1. Client initiation.
-The communication channel is initiated by a client device, just as the end to end application flows are also typically initiated by a client.
-1. Network properties sent from the network. 
-The network provides the properties to the client. The client might communicate with the network, but won't be providing network properties. 
+1. One communication channel for client initiation and network properties
```

Comment:
micro nit: Maybe "single" instead of "one".

Reply:
"One" was clear to me, but re-reading now - it's NOT clear whether that's "one" for both, or "one" for client initiation and "one" for network properties. "Single" would be clearer, but I think "single communication channel for both" would be even clearer than than.